src/components/PeopleList.tsx

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable */
+import React from 'react';
+import { Person } from '../types/Person';
+
+type Props = {
+  posts: Person[];
```

Comment:
Maybe it's better to name this prop as `people`? It not looks like posts but like array of Person

Reply:
isn't fixed